Assets/MixedRealityToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -114,16 +118,13 @@ private void SetFloorRendering()
 
         private void SetBoundaryRendering()
         {
-#if UNITY_2017_2_OR_NEWER
```

Comment:
Please put this back in.

Reply:
So things checked into the Dev_working_branch need to support pre-2017.2 builds of Unity?

I started making the change but the amount of change and configuration testing needed was so high I wanted to be sure.  It still may be easier to delay this PR until UNITY_2017_2_OR_NEWER checks aren't needed.